enterprise/server/backends/distributed/distributed.go

```diff
@@ -134,6 +134,7 @@ func (c *Cache) remoteContains(ctx context.Context, d *repb.Digest) (bool, error
 func (c *Cache) backfillReplica(ctx context.Context, d *repb.Digest, source, dest string) error {
 	log.Printf("Backfilling %q from %q => %q", d.GetHash(), source, dest)
 	r, err := c.cacheProxy.RemoteReader(ctx, source, c.prefix, d, 0)
+	defer r.Close()
```

Comment:
Typically this comes after the err checking since if err != nil then usually r is nil and defering r.Close() might give you a npe.

Reply:
Fixed!